src/test/kotlin/net/maizegenetics/phgv2/cli/CreateRefVCFTest.kt

```diff
@@ -151,5 +150,27 @@ class BuildRefVCFTest {
         val numberOfReferenceLines = bufferedReader(outFileCompressed).readLines().filter { it.startsWith("##reference") }.size
         assertEquals(1, numberOfReferenceLines)
 
+        // Verify the RefAllele for the first haplotype for each chromosome is correct
+        val genomeLines = bufferedReader(genome).readLines()
```

Comment:
Will this close the file? Do you need .use{}?

Reply:
I believe it closes it.  Checked with Zack and he believes it is fine as well.  And this is just a short junit so it will be closed after execution anyway.